Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  * If `FamilyName` has a space in it please let us know
+  
+#### **has contributor** - *person name*
+  * This is anybody who worked on this project that you would like to include 
+    in your data citation
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Provide the list of names in the order that you would like them to appear 
+    in your citation.
+  * Please include the PI / PM in this list of names
+  * Please provide names in the following format: `GivenName FamilyName`
+  * `GivenName` can include a middle name or initial
+  * If `FamilyName` has a space in it please let us know
+
+#### **has creator** - *person name*
+  * This is usually the PI and / or the person who is primarily responsible 
+    for this project
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Please provide names in the following format: `GivenName FamilyName`
+  * `GivenName` can include a middle name(s) or initial(s)
+  * If `FamilyName` has a space in it please let us know
+  
+#### **is subject of publication** - *DOI*
+  * This is a hyperlink to a publication about this project
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Provide the URL in the form of a DOI, so we can be consistent with other projects.
+  * Example: `http://doi.org/<doi_number>/<publication_id>`
+  
+#### **Title** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your project title
+    (note: Title _is not_ displayed for _data collections_)
+  
+#### **Short title** - *string*
+  * This is a simple string is currently not displayed. 
+    (note: Short title _is_ displayed for _data collections_)
+  * Please limit short titles to 30 characters or less
+  
+#### **Description** - *string*
+  * This is a simple string that will be displayed on 
+    https://knowledge.brain-map.org/ as your project description
+  
+
+## Data collection
+This sheet contains information about your data collection
+
+### **Collection** - *collection name*
+  * This is a unique human-readable identifier (i.e. handle) used to keep track 
+    of your data collection
+  * This field is controlled by the vocabulary in the `collection` column on `values` sheet
+  * If this is a new data collection, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+    and let us know.
+  * Collection names must be unique, so make sure any new data collections do 
+    not have a name that is already in the controlled vocabulary
+  * By convention, we prefer to use names in `snake_case` without `_proj` as a suffix
+  * This field corresponds to the `has output` field on the `Data collection project` tab
+    
+#### **has samples of type** - *specimen type*
```

Comment:
Add a line:   
The specimen type listed here should match the specimen/sample whose identifier is listed in the Sample Inventory template.

Reply:
Added this, but I think some of the vocabulary differs between these two templates.